src/Any/Signer.cpp

```diff
@@ -157,6 +158,27 @@ Any::Proto::SigningOutput Any::Signer::sign() const noexcept {
             }
             break;
         }
+        case TWCoinTypeVeChain: {
+            VeChain::Proto::SigningInput message;
+            parse(transaction, &message, output);
+            if (output.success()) {
+                auto veChainTransaction = VeChain::Transaction();
+                veChainTransaction.chainTag = static_cast<uint8_t>(message.chain_tag());
+                veChainTransaction.blockRef = message.block_ref();
+                veChainTransaction.expiration = message.expiration();
+                for (auto& clause : message.clauses()) {
+                    veChainTransaction.clauses.emplace_back(clause);
```

Comment:
how come for vechain we need to set all properties manually, but for cosmos it's all parsed automatically?

Reply:
It's better to move `veChainTransaction` building logic into `VeChain::Signer`, just like `Ethereum::Signer`, so we can remove this duplicate code